public/svg/logo.js

```diff
@@ -0,0 +1,39 @@
+import styled from "styled-components";
+const Logostyle = styled.svg`
+  margin: 10px 30px 0px 30px;
+  width: 75px;
```

Comment:
Even if we use this, we will anyway would have to customize those scales, to achieve the design we are aiming for and to customize those, it would take a lot of time.

Reply:
I'd prefer you do this now rather than later. It would get harder as we add more pages.
Having said that if you do still recommend doing this later, then please create an issue for this.

Reply:
I would prefer doing this at the end.